docs: improvement to “Get Started” section (#346)
bytebase/bytebase.com: content/docs/en/get-started/configure-workspace/overview.md

title: Configure the Workspace
---

Now that you have successfully deployed Bytebase. Before making any changes to databases, you should first configure the workspace.

> A single Bytebase deployment represents a single **Workspace**. At the workspace level you can configure global settings applicable to all later created projects. There are three roles at the workspace level: **Owner**, **DBA**, and **Developer**.

Reviewer: ```suggestion
A single Bytebase deployment represents a single **Workspace**. At the workspace level, you can configure global settings applicable to all projects within this workspace. There are three roles at the workspace level: **Owner**, **DBA**, and **Developer**.
```
